cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -64,24 +94,30 @@ func (w *wizard) activateTranscoder() {
 		"blockRewardCut":  {fmt.Sprintf("%v", blockRewardCut)},
 		"feeShare":        {fmt.Sprintf("%v", feeShare)},
 		"pricePerSegment": {fmt.Sprintf("%v", pricePerSegment.String())},
+		"serviceURI":      {fmt.Sprintf("%v", serviceURI)},
 		"amount":          {fmt.Sprintf("%v", amount.String())},
 	}
 
 	httpPostWithParams(fmt.Sprintf("http://%v:%v/activateTranscoder", w.host, w.httpPort), val)
+	// TODO we should confirm if the transaction was actually sent
+	fmt.Println("\nTransaction sent. Once confirmed, please restart your node.")
 }
 
 func (w *wizard) setTranscoderConfig() {
 	fmt.Printf("Current token balance: %v\n", w.getTokenBalance())
 
-	blockRewardCut, feeShare, pricePerSegment := w.promptTranscoderConfig()
+	blockRewardCut, feeShare, pricePerSegment, serviceURI := w.promptTranscoderConfig()
 
 	val := url.Values{
 		"blockRewardCut":  {fmt.Sprintf("%v", blockRewardCut)},
 		"feeShare":        {fmt.Sprintf("%v", feeShare)},
 		"pricePerSegment": {fmt.Sprintf("%v", pricePerSegment.String())},
+		"serviceURI":      {fmt.Sprintf("%v", serviceURI)},
 	}
 
 	httpPostWithParams(fmt.Sprintf("http://%v:%v/setTranscoderConfig", w.host, w.httpPort), val)
+	// TODO we should confirm if the transaction was actually sent
+	fmt.Println("\nTransaction sent. Once confirmed, please restart your node if the ServiceURI has been reset")
```

Comment:
The reset is just to allow the node to check for you if your service URI matches your publicAddr before moving forward with any other operations?

Reply:
The restart also allows the node to bind to the new port, if that has changed.